src/app/sharedUI/items/ArtworkGridItem.tsx

```diff
@@ -26,7 +27,7 @@ export const ArtworkGridItem: React.FC<ArtworkGridItemProps> = (props) => {
           }}
         />
         <Text italic variant="xs" color="black60" mt={1}>
-          {artwork.title},{" "}
+          <AvailabilityDot availability={artwork.availability || ""} /> {artwork.title},{" "}
```

Comment:
```suggestion
          <AvailabilityDot availability={artwork.availability} /> {artwork.title},{" "}
```
Lets avoid using "" and fix the type in <AvailabiltyDot/> file. Also using a switch case will help us.